apps/teams-test-app/src/components/ScrollToTop.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+
+const ScrollToTopButton = () => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  // Function to check scroll position and set visibility
+  const toggleVisibility = () => {
+    if (window.scrollY > 500) {
+      // Adjust 500 to the scroll position you deem appropriate
```

Comment:
This comment would be a good one to move to the name const you make to store this value (and probably remove reference to 500 in case that value changes in the future